Dockerfile

```
@@ -1,12 +1,17 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-storage-operator
 COPY . .
-RUN make
+RUN make && \
+    mkdir -p /tmp/build && \
+    cp /go/src/github.com/openshift/cluster-storage-operator/cluster-storage-operator-tests-ext /tmp/build/cluster-storage-operator-tests-ext && \
+    gzip /tmp/build/cluster-storage-operator-tests-ext
```

Just compress the binary where it is, why copy?

```suggestion
    gzip /go/src/github.com/openshift/cluster-storage-operator/cluster-storage-operator-tests-ext 
```

If you want to keep the source, then `gzip -k`.